zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -181,9 +80,10 @@ private[zhttp] final case class Handler[R](
   /**
    * Executes http apps
    */
-  private def unsafeRun[A](
-    http: Http[R, Throwable, A, Response[R, Throwable]],
-    a: A,
+  private def unsafeRun(
+    http: Http[R, Throwable, Request, Response[R, Throwable]],
+    a: Request,
+    msg: FullHttpRequest,
```

Comment:
Shouldn't require `msg` as argument as request already has the body.

Reply:
msg is passed as it should be fired in the channel after protocol switch